src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/categories.yml

```diff
@@ -1,10 +1,13 @@
 admin_categories_index:
-    path: /
+    path: /{categoryId}
```

Comment:
Should we, in another PR, handle other entities with parent-children data structure the same way ? I believe it might be the case for CMS Categories

Reply:
Yes absolutely, which makes me think they might have the same bug..